src/plugins/intel_cpu/CMakeLists.txt

```diff
@@ -359,7 +359,7 @@ if(BUILD_SHARED_LIBS)
             ${CMAKE_CURRENT_SOURCE_DIR}/src
             $<TARGET_PROPERTY:openvino::conditional_compilation,INTERFACE_INCLUDE_DIRECTORIES>)
 
-    target_include_directories(${TARGET_NAME}_obj SYSTEM PUBLIC $<TARGET_PROPERTY:dnnl,INCLUDE_DIRECTORIES>)
```

Comment:
I think it should be reverted.

`SYSTEM` means don't emit warnings from headers of `dnnl`

Reply:
This change helped to avoid usage of system onednn headers for CPU plugin on my local machine. Seems like it changes order of include files search as well:
https://www.intel.com/content/www/us/en/docs/dpcpp-cpp-compiler/developer-guide-reference/2024-2/isystem.html

I'll remove it if CPU plugin build still fails in CI and we'll need to manually remove system onednn.